test/schema.js

```diff
@@ -713,6 +713,59 @@ describe('JSON Schema to Data Structure', () => {
     expect(enumerations.get(2)).to.be.instanceof(NullElement);
   });
 
+  it('produces samples for enum element', () => {
+    const schema = {
+      enum: ['one', 'two'],
+      examples: ['one', 'two']
+    };
+
+    const dataStructure = schemaToDataStructure(schema);
+
+    expect(dataStructure.element).to.equal('dataStructure');
+    expect(dataStructure.content.element).to.equal('enum');
+
+    const enumerations = dataStructure.content.attributes.get('enumerations');
+
+    expect(enumerations.length).to.equal(2);
+    expect(enumerations.get(0)).to.be.instanceof(StringElement);
+    expect(enumerations.getValue(0)).to.equal('one');
+    expect(enumerations.get(1)).to.be.instanceof(StringElement);
+    expect(enumerations.getValue(1)).to.equal('two');
+
+    const samples = dataStructure.content.attributes.get('samples');
+
+    expect(samples.length).to.equal(2);
+    expect(samples.get(0).element).to.be.equal('enum');
+    expect(samples.get(0).toValue()).to.be.deep.equal('one');
+    expect(samples.get(1).element).to.be.equal('enum');
+    expect(samples.get(1).toValue()).to.be.deep.equal('two');
```

Comment:
I think it would be worth adding an additional check that `samples.get(1)`'s content is actually a string element vs a string directly. Since `toValue()` will return the string value regardless.